mlflow/experiments.py

```diff
@@ -108,3 +109,15 @@ def rename_experiment(experiment_id, new_name):
     store = _get_store()
     store.rename_experiment(experiment_id, new_name)
     print("Experiment with id %s has been renamed to '%s'." % (experiment_id, new_name))
+
+
+@commands.command("csv")
+@EXPERIMENT_ID
+@click.option("--filename_csv", type=click.STRING, required=True)
+def generate_csv_with_runs(experiment_id, filename_csv):
+    # type: (str, str) -> None
+    """
+    Generate CSV with all runs for an experiment
+    """
+    runs = fluent.search_runs(experiment_ids=experiment_id)
+    runs.to_csv(filename_csv)
```

Comment:
I think adding the `index=False` parameter seems like a good idea for this dataframe since the index numbers don't actually mean much.

Also, please add a print statement at the end with something like `print("Experiment with ID %s has been exported as a CSV to file: %s" % (experiment_id, filename))`

Reply:
Done